packages/o365_metrics/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.8.0"
+  changes:
+    - description: Add 'entra_app_registrations' data stream.
```

Based on the data stream name.
```suggestion
    - description: Add 'app_registrations' data stream.
```